### PR TITLE
Feat/dummy classifier

### LINKED ISSUE #55: Use Robust Scaler and do not drop that much outliers
Robust Scaler can handle outliers much better than Standard Scaler. Therefore try not to drop 10% quantile but rather use Robust Scaler.

### LINKED ISSUE #94: Use dummy classifier to test classification power
Try out direction classification

### LINKED ISSUE #101: Geopredict: Analyze predictive performance on different subsets
All cities in your datasets have universities. Based on Start information, predict whether a trip will
be towards, or away from the university. Analyze your predictive performance on different subsets
of your data (months, …). Do you see differences?

### LINKED ISSUE #106: Add PCA for Classification

### LINKED ISSUE #108: Add validation set to training set such that hyperparameters could be evaluated not on test set
Create Validation set from training set as 20% of whole dataset.

### LINKED ISSUE #109: Add dummy regression
Add the regression by a dummy regressor

### PR DESCRIPTION
**Pull request by weisske**
should be merged before changes in prediction package

### CODE REVIEW

**Review by MaxStroh — APPROVED** [review]
You're a freaking beast! Great job!! Nice code structuring in cli py! :) 👍 

Highly approve this pull request!!!

**Review by Speedfan94 — APPROVED** [review]
Nice changes :-)

**Review by mulko93 — APPROVED** [review]
reviewed im team meeting. changes look appropriate

**Review by Speedfan94 — APPROVED** [review]
approved

**Comment by Speedfan94** [comment]
Fixed tasks from Live-Session:
fixed robust scaler-test
fixed subset evauluations

**Review by MaxStroh — APPROVED** [review]
Additional fixes approved! Lokks good :)

**Review by Speedfan94 — APPROVED** [review]
corrections from live session
